integration-tests/bats/sql-backup.bats

```diff
@@ -17,21 +17,63 @@ teardown() {
 }
 
 @test "sql-backup: dolt_backup add" {
-    run dolt sql -q "call dolt_backup('add', 'hostedapidb-0', 'file:///some_directory')"
-    [ "$status" -ne 0 ]
-    run dolt sql -q "CALL dolt_backup('add', 'hostedapidb-0', 'file:///some_directory')"
-    [ "$status" -ne 0 ]
+    mkdir the_backup
+    run dolt sql -q "call dolt_backup('add', 'hostedapidb-0', 'file://./the_backup')"
+    [ "$status" -eq 0 ]
+    run dolt backup -v
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ "the_backup" ]] || false
+}
+
+@test "sql-backup: dolt_backup ass cannot add remote with address of existing backup" {
+    mkdir bac1
+    dolt backup add bac1 file://./bac1
+    run dolt remote add rem1 file://./bac1
+    [ "$status" -eq 1 ]
+    [[ "$output" =~ "address conflict with a remote: 'bac1'" ]] || false
+}
+
+@test "sql-backup: dolt_backup remove" {
+    mkdir bac1
+    dolt backup add bac1 file://./bac1
+    run dolt backup -v
+    [ "$status" -eq 0 ]
+    [ "${#lines[@]}" -eq 1 ]
+    [[ "$output" =~ "bac1" ]] || false
+
+    dolt backup remove bac1
+
+    run dolt backup -v
+    [ "$status" -eq 0 ]
+    [ "${#lines[@]}" -eq 0 ]
+
+}
+
+@test "sql-backup: dolt_backup remove cannot remove non-existent backup" {
+    run dolt backup -v
+    [ "$status" -eq 0 ]
+    [ "${#lines[@]}" -eq 0 ]
+    [[ ! "$output" =~ "bac1" ]] || false
+
+    run dolt backup remove bac1
+    [ "$status" -eq 1 ]
+    [[ "$output" =~ "error: unknown backup: 'bac1'" ]] || false
 }
 
 @test "sql-backup: dolt_backup rm" {
-    run dolt sql -q "call dolt_backup('rm', 'hostedapidb-0')"
-    [ "$status" -ne 0 ]
-    run dolt sql -q "CALL dolt_backup('rm', 'hostedapidb-0')"
-    [ "$status" -ne 0 ]
-    run dolt sql -q "call dolt_backup('remove', 'hostedapidb-0')"
-    [ "$status" -ne 0 ]
-    run dolt sql -q "CALL dolt_backup('remove', 'hostedapidb-0')"
-    [ "$status" -ne 0 ]
+    mkdir bac1
+    dolt backup add bac1 file://./bac1
+    run dolt backup -v
+    [ "$status" -eq 0 ]
+    [ "${#lines[@]}" -eq 1 ]
+    [[ "$output" =~ "bac1" ]] || false
+
+    dolt backup rm bac1
+
+    run dolt backup -v
+    [ "$status" -eq 0 ]
+    [ "${#lines[@]}" -eq 0 ]
+    [[ ! "$output" =~ "bac1" ]] || false
 }
 
```

Comment:
(minor) Might be nice to add in a quick test for the error case when an http/https URL is provided.